### PR TITLE
Bbq10 kb fix

### PR DESCRIPTION
**Pull request by caveman99**

### CODE REVIEW

File: src/input/kbI2cBase.cpp [hunk 1]
```diff
@@ -78,21 +78,49 @@ int32_t KbI2cBase::runOnce()
                     e.inputEvent = meshtastic_ModuleConfig_CannedMessageConfig_InputEventChar_BACK;
                     e.kbchar = key.key;
                     break;
-                case 0x12: // sym shift+2
-                    e.inputEvent = meshtastic_ModuleConfig_CannedMessageConfig_InputEventChar_UP;
-                    e.kbchar = 0xb5;
+                case 'e': // sym e
+                case '2': // sym shift+2
```
Comment (mverch67) [inline src/input/kbI2cBase.cpp:82]: should probably be 'E'

File: src/input/kbI2cBase.cpp [hunk 2]
```diff
@@ -78,21 +78,49 @@ int32_t KbI2cBase::runOnce()
...
+                        e.inputEvent = ANYKEY;
+                        e.kbchar = key.key;
+                    }
                     break;
-                case 0x18: // sym shift+8
-                    e.inputEvent = meshtastic_ModuleConfig_CannedMessageConfig_InputEventChar_DOWN;
-                    e.kbchar = 0xb6;
+                case 'x': // sym x
+                case '8': // sym shift+8
+                    if (is_sym) {
```
Comment (mverch67) [inline src/input/kbI2cBase.cpp:93]: should probably be 'X'

**Comment by caveman99** [comment]
Right now the sym state will not reset itself after the 2nd keypress, the device will stay in SYM mode till you press the button again. Much like Capslock. I will add a reset when i do the final cleanup.

**Comment by caveman99** [comment]
Do we have a final verdict if this can go in?

**Pull request merged**